canonical-websites/build.snapcraft.io
# Snaps details API

Part of #655 

To know if snap was released to stable channel we are proxying [CPI snaps details API](https://search.apps.ubuntu.com/docs/#details)

This PR simply exposes `/api/snaps/details/:name` route in BSI to be able to talk to CPI API cross origin.

### QA:

To test this PR run local BSI server and then try to go to API URL in the browser, for example: http://localhost:8000/api/snaps/details/core

You can also pass `fields`, `channel` query params that will be passed to CPI API:
http://localhost:8000/api/snaps/details/core?channel=beta&fields=name,revision

API should return a JSON response with details of given snap.

You can change last part of the URL to any public snap name. If snap doesn't exist API should return 404.
